tools/schemacode/bidsschematools/tests/test_rules.py

```diff
@@ -50,8 +50,8 @@ def test_entity_rule(schema_obj):
     json_rule = rules._entity_rule(rule, schema_obj)
     assert json_rule == {
         "regex": (
-            r"(?:sub-(?P<subject>[0-9a-zA-Z]+)/)?"
-            r"(?:ses-(?P<session>[0-9a-zA-Z]+)/)?"
+            r"(?:sub-(?P<subject>[0-9a-zA-Z+]+)/)?"
+            r"(?:ses-(?P<session>[0-9a-zA-Z+]+)/)?"
```

Comment:
I am afraid that is not all ...  I will push a few commits on that end in a few minutes (I hope you don't mind).
Some other might need adjustment and I even start feeling that we might need to come up with some term (like "literal" but there might be better) to encompass "alphanumeric" and `+`.

Reply:
@yarikoptic Note this is a regression test that shows the specific output of a specific synthetic rule.

Reply:
my comment is not really about this test -- I meant that changes in this PR (just this test) aren't sufficient. pushed now